pkg/api/controller.go

```diff
@@ -156,7 +156,9 @@ func (c *Controller) Run() error {
 		return err
 	}
 
-	monitoring.SetServerInfo(c.Metrics, c.Config.Commit, c.Config.BinaryType, c.Config.GoVersion, c.Config.Version)
+	monitoring.SetServerInfo(c.Metrics, c.Config.Commit, c.Config.BinaryType, c.Config.GoVersion,
```

Comment:
is this line split because of linter?

Reply:
Yes, the line has more than 111 characters and I had to split it.